.travis.yml

```diff
@@ -3,22 +3,39 @@ language: python
 
 matrix:
     include:
-        - python: "3.4"
+        - python: "3.5"
```

Comment:
dropping 3.4 / ubuntu trusty testing for linux is not in scope of this PR.

if 3.4 / trusty does not work for ppc, just leave it away there.